docs-kits/kits/PCF Exchange Kit/page_software-operation-view.md

```diff
@@ -4,6 +4,7 @@ title: Operation View
 description: 'PCF Exchange KIT'
 sidebar_position: 4
 ---
->**Note**
->
-> At the moment there is no open source soultion available implementing PCF exchange. Therefore no central operation information is provided here.
+In the past, the Catena-X ecosystem for PCF data exchange consisted exclusively of commercial applications (so-called COTS apps). For release 24.05, with the Simple Data Exchanger (SDE), for the first time an open source solution is announced to support the CX PCF Use Case Standard (data model, API,...). Further information regarding the SDE (usage, configuration and deployment) are published on GitHub:
```

Comment:
I would suggest to adjust the sentence to:

"In addition to certified commercial solutions, there is now also an open-source reference implementation available for the first time..."